downstream/modules/builder/proc-customize-ee-image.adoc

```diff
@@ -56,7 +58,7 @@ Since this example uses the community version of `kubernetes.core` and not a cer
 +
 [subs=+quotes]
 ----
-$ ansible-builder build -t registry.redhat.io/_[username]_/_new-ee_
+$ ansible-builder build -t _[username]_/_new-ee_
```

Comment:
Somewhere after step 5 in Chapter 4.1, the numbering gets out of sequence. In the build, step 5 is followed by "a", but not as substeps and then the next step starts renumbering beginning with 1: 

![Screenshot from 2023-10-23 18-41-25](https://github.com/ansible/aap-docs/assets/73364088/e2dd3e84-d31a-4f25-97f3-7b6fc36d2a7a)

Reply:
This has been fixed, note I changed Step 5.a to be a bulleted item since single steps should be bullets and not numbers.